src/helpers/nsfwClassifier.js

```diff
@@ -1,19 +1,20 @@
-import { nsfwThresholds } from '@/helpers/constants/nsfwThresholds';
+import { nsfwThresholds } from "@/helpers/constants/nsfwThresholds";
 
-export const mimetypeExemptions = ['image/svg+xml'];
+export const mimetypeExemptions = ["image/svg+xml"];
 
 // for a local setup, typically use: http://localhost:8080/classify
-const nsfwApiEndpoint = process?.env?.NSFW_API || 'https://api.ipfs-search.com/v1/nsfw/classify/';
+const nsfwApiEndpoint =
+  import.meta.env.VITE_NSFW_API || "https://api.ipfs-search.com/v1/nsfw/classify/";
 
 export function classify({ hash, mimetype }) {
   if (mimetypeExemptions.includes(mimetype)) return Promise.resolve({});
   return fetch(`${nsfwApiEndpoint}${hash}`, {
-    method: 'GET',
+    method: "GET",
     headers: {
-      Accept: 'application/json',
+      Accept: "application/json",
     },
-  })
-    .then((response) => response.json());
+    mode: "no-cors",
```

Comment:
AFAIK you will need CORS here, otherwise you won't get results back. NSFW results should have CORS headers unless they are errors (e.g. 404's do not seem to have them).